{"name": "test_verify_login_with_valid_credentials", "status": "passed", "start": 1722047876375, "stop": 1722047877286, "uuid": "ac5f9c4b-c894-4dcc-b836-27da7b3bc1ca", "historyId": "14f2072526db18090e3cb1e2c1ce29e3", "testCaseId": "14f2072526db18090e3cb1e2c1ce29e3", "fullName": "test.test_Login.TestLogin#test_verify_login_with_valid_credentials", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Raunak"}, {"name": "thread", "value": "10708-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_Login"}]}